azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -146,9 +240,167 @@ func TestAccAzureRMMonitorActionGroup_webhookReceiver(t *testing.T) {
+					resource.TestCheckResourceAttr(resourceName, "itsm_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "azure_app_push_receiver.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.0.service_uri", "http://example.com/alert"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "voice_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "logic_app_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "azure_function_receiver.#", "0"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAzureRMMonitorActionGroup_automationRunbookReceiver(t *testing.T) {
+	resourceName := "azurerm_monitor_action_group.test"
+	ri := tf.AccRandTimeInt()
+	config := testAccAzureRMMonitorActionGroup_automationRunbookReceiver(ri, testLocation())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMMonitorActionGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMMonitorActionGroupExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "itsm_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "azure_app_push_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.automation_account_id", "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.runbook_name", "my runbook"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.webhook_resource_id", "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"),
+					resource.TestCheckResourceAttrSet(resourceName, "automation_runbook_receiver.is_global_runbook"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.service_uri", "https://s13events.azure-automation.net/webhooks?token=randomtoken"),
```

Comment:
heads up the `Import` step below will check these values match the ones in the config - as such we shouldn't need to compare this here :)

Reply:
@tombuildsstuff Thank you so much for all the helpful support on my first Terraform provider PR.  I'm still tracing through how Import works in the terraform helpers but will attend to this soon.